### DISCUSSION TITLE
Testing: Ability to fake listeners

### DISCUSSION THREAD

**Discussion by Braunson** [laravel/framework#47118] [Ideas]
The ability to do something along the lines of `Listener::fake()` or specify directly one or more listeners via `Listener::fake([FooListener::class, BarListener::class]);`

**Comment by henzeb** [comment]
We already have Event::fake. Or is that not what you meant? 

https://laravel.com/docs/10.x/events#testing

**Reply by Braunson** [reply]
I tried that but it didn't seem to work for a Listener..
